src/pages/components/carousel.mdx

```diff
@@ -50,6 +50,45 @@ Carousel should display content consistently throughout the same carousel. For e
 3. **Carousel counter:** Displays the current view number and the total view number in this carousel. The total number may change based on the breakpoint being viewed.
 4. **Next icon button:** Advances the carousel to the next view, containing additional displayed content.
 
+## Usage guidance
+
+### When should I use a carousel?
+
+- Use when you have multiple related content frames but screen real estate is limited.
+- Use when you have exploratory content that isn’t necessary but helpful for the user.
+- Use when you want to tease the user with additional related content.
+
+### When should I not use a carousel?
+
+- Do not use when content frames are not related.
+- Do not use when users need to consume all carousel content – users are never required to advance the slides to see all content.
+
+### Content
+
+- If you are using cards as your content, the recommended number of cards is a minimum of 5 cards and a maximum of 15 cards to avoid excessive clicking for the user.
+  Make sure all content in the carousel is related.
+
+- Do not add content that is essential to a carousel. This includes navigational or wayfinding content links that would lead the user to a key resource.
+
+- Prioritize the most important content at the beginning of the carousel, so that it is visible without needing to advance the carousel view.
+
+- Carousel content should be simple and concise.
+
+- Common content used in carousels includes use cases, news articles, client testimonials, and quotes.
+
```

Comment:
```suggestion
- If you are using cards as your content, the recommended number of cards is a minimum of 5 cards and a maximum of 15 cards to avoid excessive clicking for the user.
```

Moving this down since it's specific to `cards`